unit-tests/func/reflectivity/l500/test-auto-set-options.cpp

```diff
@@ -0,0 +1,35 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+//#test:tag ir-reflectivity
```

Comment:
Why do you keep calling it "IR-" reflectivity?
It's the reflectivity that we care about, no? IR is the implementation? Do we foresee another kind of reflectivity?

Reply:
On Ofir script and on the ticket it say IR-Reflectivity, so that's the name I gave it.

Reply:
For the tag, though, I think reflectivity is good enough

Reply:
(matching the tag in the tests you have below... :)

Reply:
👍